CHANGELOG.md

```diff
@@ -1,4 +1,6 @@
-# Unreleased
+# 1.0.0-alpha.5
+
+* Feat: Ability to configure url for on-premise server (#17)
```

Comment:
```suggestion
# Unreleased

* Feat: Ability to configure url for on-premise server (#17)
```

Reply:
since we change the versioning during releasing

Reply:
Right. Done.